masonry/src/testing/helper_widgets.rs

```diff
@@ -335,8 +335,8 @@ impl Widget for ReplaceChild {
         self.child.accessibility(ctx);
     }
 
-    fn children(&self) -> SmallVec<[WidgetRef<'_, dyn Widget>; 16]> {
-        self.child.widget().children()
+    fn children_ids(&self) -> SmallVec<[WidgetId; 16]> {
+        todo!()
```

Comment:
Did you want to handle that `todo!` in this PR?

Reply:
Not sure why I kept it. Will address in smaller PR.